data_splitting.py

```diff
@@ -105,6 +105,7 @@ def process_data_splitting_scenario(scenario):
   
     # Populate nodes
     node_list = []
+    nodeId=1
```

Comment:
@celinejacques could you reformat it to `node_id` to enforce the snake_case style?